scss/forms/_form-control.scss

```diff
@@ -45,6 +44,13 @@
     }
   }
 
+  // Add some height to date inputs on IOS
+  // https://github.com/twbs/bootstrap/issues/23307
+  &::-webkit-date-and-time-value {
```

Comment:
I suppose this in not in the specs, right?

Reply:
Nope. This is a bug in Safari's default CSS (see https://opensource.apple.com/source/WebCore/WebCore-7602.4.6/css/html.css.auto.html, input::-webkit-inner-spin-button)

This bug has been reported here and I've just suggested the fix on that issue: https://bugs.webkit.org/show_bug.cgi?id=198959

Reply:
Maybe we should add a TODO comment then?

Reply:
Done